packages/angular/src/generators/library/lib/update-tsconfig.ts

```diff
@@ -14,9 +19,11 @@ function updateRootConfig(host: Tree, options: NormalizedSchema) {
     }
 
     c.paths[options.importPath] = [
-      `${getWorkspaceLayout(host).libsDir}/${
-        options.projectDirectory
-      }/src/index.ts`,
+      joinPathFragments(
+        getWorkspaceLayout(host).libsDir,
+        options.projectDirectory,
+        '/src/index.ts'
+      ),
```

Comment:
this is a big deal because my tsconfig.base was configured with absolute routes due to the empty `libsDir` and the paths are not detected